### PR TITLE
Feature display client

### PR DESCRIPTION
**Pull request by alaindavis90**
User name can now be displayed on profile page.

### CODE REVIEW

File: client/src/components/UserPanel.js [hunk 1]
```diff
@@ -58,12 +59,12 @@ function UserPanel(props) {
         />
       </Grid>
       <Grid item id="name" className={classes.item}>
-        <Typography className={classes.primaryText}>John Doe</Typography>
-        <Typography className={classes.secondaryText}>@johndoe</Typography>
+        <Typography className={classes.primaryText}>{name}</Typography>
+        <Typography className={classes.secondaryText}>@{name}</Typography>
```
Comment (skassam21) [inline client/src/components/UserPanel.js]: this should be username right?

File: server/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "http-errors": "~1.6.3",
     "joi": "^14.3.1",
     "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.14",
```
Comment (skassam21) [inline server/package.json]: Can we also update lodash on the front-end ^4.17.14 and run yarn install. lower versions actually had security problem

File: client/src/components/UserPanel.js (hunk 1 above)
Comment (skassam21) [reply client/src/components/UserPanel.js]: are we adding that field?

Comment (gsmandur) [reply client/src/components/UserPanel.js]: yes I was going to bring that up during the sprint planning. We will have to refactor that sooner rather ran later.

File: client/src/pages/Login.js [hunk 3]
```diff
@@ -54,11 +56,11 @@ class Login extends Component {
         console.log("token from local storage = ", localStorageToken);
 
         //direct user to profile page
-        this.props.history.replace("/profile");
+        this.props.history.push('/profile', { id: response.data._id });
+
       })
       .catch(error => {
         console.log("ERROR:", error);
-        this.setState({ errorMessage: error.response.data });
```
Comment (gsmandur) [inline client/src/pages/Login.js]: Any reason this error message was removed?

**Pull request merged**